temporal-sdk/src/test/java/io/temporal/workflow/shared/TestActivities.java

```diff
@@ -94,6 +94,11 @@
     Map<String, Duration> activity2();
   }
 
+  @ActivityInterface
```

Comment:
I think there is no reason to grow `TestActivities` interface, just move declaration to your test.

Reply:
Used in more than 1 test, so I moved it here.